src/Duende.AccessTokenManagement/ClientCredentialsTokenManagementBuilder.cs

```diff
@@ -31,6 +31,13 @@ public ClientCredentialsTokenManagementBuilder(IServiceCollection services)
     public ClientCredentialsTokenManagementBuilder AddClient(string name, Action<ClientCredentialsClient> configureOptions)
     {
         _services.Configure(name, configureOptions);
+        _services.PostConfigure<ClientCredentialsClient>(name, client =>

```

Comment:
Are there other circumstances where we could check for this configuration error, such as in a code flow?

Reply:
Probably, but the scope of this PR was intended to fix an existing but misleading configuration check.